app/controllers/concerns/api/v1/participant_actions.rb

```diff
@@ -22,7 +22,7 @@ def resume
       end
 
       def change_schedule
-        service = ChangeSchedule.new(action_params)
+        service = ChangeSchedule.new(action_params.merge!(allow_change_to_from_frozen_cohort: true))
```

Comment:
I think we might need to merge both prs especially regarding https://github.com/DFE-Digital/early-careers-framework/pull/4963/files#diff-dceb8eaba398eba134e24e36e018bf28066f0a45746bfaaabcfb48cc7ab88b12R238 let's chat tomorrow :)